ska_trend/wrong_box_acq/task_schedule.cfg

```diff
@@ -0,0 +1,56 @@
+# Configuration file for task_schedule.pl to run astromon jobs
+
+subject           Wrong Box Acq Anom
+timeout           360               # Default tool timeout
+heartbeat_timeout 30000             # Maximum age of heartbeat file (seconds)
+iterations        1                 # Run once then shut down task_schedule
+print_error       1                 # Print full log of errors
+disable_alerts    0                 # Don't disable alerts since this jobs runs just once/day
+loud              0                 # Run loudly or quietly (production mode)
+
+# Data files and directories.  The *_dir vars can have $ENV{} vars which
+# get interpolated.  (Note lack of task name after TST_DATA because this is just for test).
+
+data_dir     $ENV{SKA_DATA}/wrong_box_trend      # Data file directory
```

Comment:
I have some ambiguity here, with 'wrong_box_trend' used in the task schedule, 'wrong_box_acq' used for the module, and 'wrong_box_anom' used for the draft page I showed at SSAWG.  So just let me know what is preferred and I'll make it all match.

Reply:
Per our conversation today, I'll just update everywhere to be wrong_box_anom.